init.rb

```diff
@@ -28,17 +28,17 @@
   description 'Plugin to generate and use issue templates for each project to assist issue creation.'
   version '0.1.1'
   author_url 'http://twitter.com/akiko_pusu'
-  requires_redmine :version_or_higher => '2.5'
+  requires_redmine version_or_higher: '2.5'
   url 'https://bitbucket.org/akiko_pusu/redmine_issue_templates'
 
-  menu :admin_menu, :redmine_issue_templates, { :controller => 'global_issue_templates', :action => 'index' },
-       :caption => :global_issue_templates
+  menu :admin_menu, :redmine_issue_templates, { controller: 'global_issue_templates', action: 'index' },
+       caption: :global_issue_templates
 
   project_module :issue_templates do
-    permission :edit_issue_templates, {:issue_templates => [:new, :edit, :destroy, :move]}
-    permission :show_issue_templates, {:issue_templates => [:index, :show, :load, :set_pulldown]}
+    permission :edit_issue_templates, { issue_templates: [:new, :edit, :destroy, :move] }
+    permission :show_issue_templates, { issue_templates: [:index, :show, :load, :set_pulldown] }

```

Comment:
__[rubocop]__
* Redundant curly braces around a hash parameter.
* Line is too long. [96/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)